guides/release/routing/defining-your-routes.md

```diff
@@ -39,19 +39,19 @@ Router.map(function() {
 });
 ```
 
-Inside your templates, you can use [`{{link-to}}`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/link-to?anchor=link-to) to navigate between
+Inside your templates, you can use [`<LinkTo>`](https://www.emberjs.com/api/ember/release/classes/Ember.Templates.helpers/methods/link-to?anchor=link-to) to navigate between
 routes, using the name that you provided to the `route` method.
```

Comment:
Do we need to mention performance cautions here?

Reply:
No, we shouldn't advise against using `LinkTo`, there really isn't a reasonable alternative yet.